src/lava/magma/runtime/message_infrastructure/message_infrastructure_interface.py

```diff
@@ -42,4 +47,6 @@ def actors(self) -> ty.List[ty.Any]:
 
     @abstractmethod
     def channel_class(self, channel_type: ChannelType) -> ty.Type[Channel]:
+        """Given the Channel Type, Return the Channel Implementation to
```

Comment:
ChannelType

Why capitalize Return & Implementation?

Is that a proper noun defined anywhere?